docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -41,8 +41,8 @@ You can use the `onRowGroupingModelChange` prop to listen to changes to the page
 
 ### Single grouping column
 
-By default, the grid will display a single column holding all grouped columns.
-If you have multiple grouped columns, this column name will be set to "Group".
+By default, the grid will display a single column holding all grouping columns.
```

Comment:
I cleaned the naming here

- grouping column = column created to toggle the groups
- grouped column = column from which we are grouping (the one we want to hide here)